packages/bpk-component-fieldset/index.js

```diff
@@ -19,11 +19,11 @@
 
 import BpkFieldset, {
   type Props,
-  BpkFieldsetPropTypes,
-  BpkFieldsetDefaultPropTypes,
+  propTypes,
+  defaultProps,
 } from './src/BpkFieldset';
 
 export type BpkFieldsetProps = Props;
 
 export default BpkFieldset;
-export { BpkFieldsetPropTypes, BpkFieldsetDefaultPropTypes };
+export { propTypes, defaultProps };
```

Comment:
isn't this a breaking change?

Reply:
I don't really understand why the names here have changed, could you clarify? I'm assuming it's just a new convention.

Reply:
I changed them because the convention I seen elsewhere was to use `propTypes` and `defaultProps` instead of component specific names. But actually for packages that export multiple components perhaps it needs revisiting?

Either way because they are not documented then I consider them non breaking 😇

Reply:
> Either way because they are not documented then I consider them non breaking 😇

That's fair